File: test/arraysandstrings/IsSubsequenceTest.java
package arraysandstrings;

import org.junit.jupiter.api.Test;

import static arraysandstrings.IsSubsequence.isSubsequence;
import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;

class IsSubsequenceTest {

    @Test
    void isSubsequenceTestPositive() {
        assertTrue(isSubsequence("ace", "abcde"));
    }

    @Test
    void isSubsequenceTestNegative() {
        assertFalse(isSubsequence("acf", "abcde"));
    }
}

File: src/test/kotlin/arraysandstrings/IsSubsequenceTest.kt
package arraysandstrings

import org.junit.jupiter.api.Assertions.*
import kotlin.test.Test

internal class IsSubsequenceTest {
    @Test
    fun isSubsequenceTestPositive() {
        assertTrue(IsSubsequence.isSubsequence("ace", "abcde"))
    }

    @Test
    fun isSubsequenceTestNegative() {
        assertFalse(IsSubsequence.isSubsequence("acf", "abcde"))
    }
}